# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../nvidia/module-install-26.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- kernel/Makefile.kbuild.orig	2012-03-17 22:55:51.000000000 +0100
+++ kernel/Makefile.kbuild	2012-04-10 23:54:38.063139448 +0200
@@ -257,8 +257,7 @@
 
 module-install: suser-sanity-check module
 	@mkdir -p $(MODULE_ROOT)/video; \
-	install -m 0664 -o root -g root $(MODULE_OBJECT) $(MODULE_ROOT)/video; \
-	PATH="$(PATH):/bin:/sbin" depmod -ae;
+	install -m 0664 -o root -g root $(MODULE_OBJECT) $(MODULE_ROOT)/video;
 
 #
 # This target builds, then installs, then creates device nodes and inserts
@@ -266,7 +265,6 @@
 #
 
 package-install: module-install rmmod-sanity-check
-	@PATH="$(PATH):/bin:/sbin" modprobe $(MODULE_NAME) && \
 	echo "$(MODULE_OBJECT) installed successfully.";
 
 #
